package.json

```diff
@@ -201,7 +202,8 @@
             ".v": "v run",
             ".vsh": "v run",
             ".sass": "sass --style expanded",
-            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt"
+            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
+            ".less": "cd $dir && lessc $fileName $fileNameWithoutExt.css"
```

Comment:
No need for this line of code, as less is pre-defined in VS Code: https://github.com/microsoft/vscode/tree/master/extensions/less

Reply:
Got it. I've removed the duplicate line.